package.json

```diff
@@ -98,7 +98,7 @@
   "engines": {
     "npm": "please-use-yarn",
     "node": ">=18",
-    "pnpm": "8.7.0"
+    "pnpm": ">=8.7.0"
```

Comment:
Is this fine or should I use this specific version of `pnpm`? I had a more recent one installed.

Reply:
I was hoping we could pin the pnpm version, just like we used to do with yarn

Reply:
So I should use 8.7.0? Why are we not using a more recent version? 
Anyway I can switch, no problem, I guess it's safer against weird issues if we all use the same version.